.github/workflows/publish.yml

```diff
@@ -31,18 +65,29 @@ jobs:
         uses: actions-rs/toolchain@v1
         with:
           toolchain: nightly-2022-07-22
+          target: ${{ matrix.build.target }}
+          components: rust-src
           override: true
 
-        # Workaround to resolve link error with C:\msys64\mingw64\bin\libclang.dll
+      - name: CUDA toolchain
+        uses: Jimver/cuda-toolkit@v0.2.6
+        if: runner.os == 'Linux' || runner.os == 'Windows'
+
+      - name: install libdbus-1-dev and pkg-config (Linux)
+        run: sudo apt -y --no-install-recommends install libdbus-1-dev pkg-config
+        if: runner.os == 'Linux'
+
+      # Workaround to resolve link error with C:\msys64\mingw64\bin\libclang.dll
       - name: Remove msys64
         run: Remove-Item -LiteralPath "C:\msys64\" -Force -Recurse
         if: runner.os == 'Windows'
 
-      - name: install webkit2gtk (ubuntu only)
-        if: runner.os == 'Linux'
+      # libayatana does not work for ubuntu20.04 -> https://github.com/subspace/subspace-desktop/runs/7581578859?check_suite_focus=true
```

Comment:
Why not? I have sent you a commit a few days ago that fixes it by simply updating a dependency. Specifically this one: https://github.com/subspace/subspace-desktop/commit/b4a96bea91157422cb5987f1f5a751ad9872d6cf

Legacy `libappindicator3` conflicts with `libayatana-appindicator3-1` and will likely make users choose either they have Subspace Desktop or other apps.